.gitignore

```diff
@@ -1,54 +1,35 @@
-# --------------------
-# OSX Files
-# --------------------
+### SublimeText ###
+*.sublime-workspace
+
+### OSX ###
 .DS_Store
 .AppleDouble
 .LSOverride
 Icon
+
+# Thumbnails
 ._*
+
+# Files that might appear on external disk
 .Spotlight-V100
 .Trashes
 
-# --------------------
```

Comment:
Meh, its up to you but I'd vote yes. IMO its in our best interest to make contribution as easy as possible no matter what env/IDE you use. If you want to tune for our use case that is fair too. Just my two cents.

Just to give you all the background, this is a very small subset that was pulled from Github's gitignore repo
https://github.com/github/gitignore